lib/connect-session-sequelize.js

```diff
@@ -23,16 +23,14 @@ util.inherits(SequelizeStoreException, Error);
 
 module.exports = function SequelizeSessionInit(Store) {
   function SequelizeStore(options) {
-    options = options || {};
+    this.options = options = options || {};
```

Comment:
simpler would be: ` this.options = options || {};` - then you'd just use everywhere in this constructor `this.options` instead of `options`.